### DISCUSSION TITLE
Help getting the initial json file able to be found by the GarminHomeAssistant online editor.

### DISCUSSION THREAD

**Discussion by mrcaseyrose** [house-of-abbey/GarminHomeAssistant#204] [Q&A]
I've followed the guide but for the life of me I can't get the json file I put in my www file to be found by the GarminHomeAssistant editor. Doing the troubleshooting steps it says the API is running and the MENU is available. The download button is grayed out and cannot be clicked. 

On my watch, it says API available, menu unavailable.

**Comment by Zlate87** [comment]
Can you provide some more info?
Are you able to access the file in a browser?

**Reply by philipabbey** [reply]
Agreed, make sure you know how to match the URL of your menu file (as used by a browser) to the file location you chose in www.
